### PR TITLE
updates Localytics to v4

### PR DESCRIPTION
**Pull request by ladanazita**
Localytics bumped to 4.0. And updates the following methods:
If you were previously changing the default session timeout interval, use the new `setOptions` method.

```
[Localytics setSessionTimeoutInterval:30];
```

to

```
[Localytics setOptions:@{@"session_timeout": @30}];
```

and if you were previously changing the default session timeout interval, use the new `setOptions` method.

```
[Localytics handlePushNotificationOpened:userInfo];
```

to 

```
[Localytics handleNotification:userInfo];
```

If merged, this closes issue #9

### CODE REVIEW

**Comment by ladanazita** [comment]
@wcjohnson11 👀

File: Example/Podfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
+use_frameworks!
```
Comment (f2prateek) [inline Example/Podfile]: Is this needed?

Comment (wcjohnson11) [inline Example/Podfile]: I believe it is since they've updated to a dynamic framework for 4.0

Comment (ladanazita) [inline Example/Podfile]: Yeah its needed

Comment (f2prateek) [inline Example/Podfile]: That doesn't mean this line is needed though? This line is only needed if you depend on a Swift pod IIRC which this app doesn't

Comment (ladanazita) [inline Example/Podfile]: It failed without the `use_frameworks!`

**Comment by ladanazita** [comment]
@f2prateek removed `use_frameworks!` 

Let me know what you think!

**Pull request merged**